scbw/vnc.py

```diff
@@ -1,17 +1,20 @@
 import os
 import shutil
 
+def find_vnc_executable() -> str:
+    for name in ["vnc-viewer", "vncviewer"]:
+        path = shutil.which(name)
+        if path:
+            return path
+    raise Exception("Neither 'vnc-viewer' nor 'vncviewer' was found. Check your PATH.")
```

Comment:
Better than before, maybe you could add a VNCVIEWER env variable, to support "wildly" named vnc viewers.